bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/TypedescValueImpl.java

```diff
@@ -52,6 +52,7 @@ public class TypedescValueImpl implements  TypedescValue {
     final Type type;
     final Type describingType; // Type of the value describe by this typedesc.
     public MapValue[] closures;
+    public MapValue annotations;
```

Comment:
What happens if someone tries to retrieve the annotations via the type now?

Reply:
?

Reply:
Can you provide an example of this? Is it possible to access annotations from a type?

Reply:
In Java code. `io.ballerina.runtime.internal.types.BAnnotatableType#getAnnotations`.

Reply:
This returns a null value. And also this is an internal method and can not be utilized by external parties. Right?

Reply:
`io.ballerina.runtime.api.types.AnnotatableType#getAnnotations`

Reply:
Even if it was only used within lang, should be consistent.

Reply:
Can we set the same annotations to the type?

Reply:
Fixed the issue by setting the annotation to the type and utilizing the type to retrieve the annotations(Except using annotations in the typedesc)